src/site/filters/liquid.js

```diff
@@ -21,6 +21,37 @@ module.exports = function registerFilters() {
   liquid.filters.humanizeTimestamp = dt =>
     moment.unix(dt).format('MMMM D, YYYY');
 
+  liquid.filters.formatMobileTableData = entity => {
```

Comment:
It might be worth adding a comment above this line describing the output of this function (e.g. `[{ caption: String, rows: [{ label: String, value: String }] }]`) since the data undergoes quite the transformation 🙂